src/Engines/thinEngine.ts

```diff
@@ -3627,7 +3627,7 @@ export class ThinEngine {
         this._activeChannel = channel;
         const target = this._getTextureTarget(internalTexture);
         if (needToBind) {
-            this._bindTextureDirectly(target, internalTexture, isPartOfTextureArray);
+            this._bindTextureDirectly((internalTexture && internalTexture.isMultiview) ? this._gl.TEXTURE_2D_ARRAY : target, internalTexture, isPartOfTextureArray);

```

Comment:
Why not doing it inside the getTextureTarget?

Reply:
It was there before, but it makes sense :-)

let me fix that